Графика для всех страниц

Kirill-hippo opened the pull request:
…ые из-ния

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2135219/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2135219-device-35/3/

Kirill-hippo committed:
Доб-ны из-ния в файлы: index.html и catalog.html. Подключены контентные из-ния

avilona reviewed (commented):
1. Цифру два, которая у пункта Корзина в меню нужно будет сделать в разметке, экспортировать как изображение неверно

![image](https://user-images.githubusercontent.com/12893725/197390178-e96fc21c-4d80-4808-8cf1-7bf3455f8893.png)

2. Машинку - отдельно, фон - при помощи css, иначе не получится реализовать переполнение в это блоке

![image](https://user-images.githubusercontent.com/12893725/197390237-bcee58a6-cd3f-4563-ae04-8f2fe5d6cf43.png)

Kirill-hippo committed:
Исправил ошибки по дз

1. удалил из папки images лишнюю картинку цифра 2 у корзины с товарами
2. загрузил в папку images картинку  delivery-car.svg в верном формате (без фона)
